swagger.yaml

```diff
@@ -4289,14 +4142,79 @@ definitions:
         type: string
         description: JSON-formatted string containing the usage information by feature name
         x-go-custom-tag: gorm:"type:text"
+        
+  infraenv-create-params:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the InfraEnv.
+      proxy:
+        $ref: "#/definitions/proxy"
+      additional_ntp_sources:
+        type: string
+        description: A comma-separated list of NTP sources (name or IP) going to be added to all the agents.
+      ssh_authorized_key:
+        type: string
+        description: SSH public key for debugging the installation.
+      pull_secret:
+        type: string
+        description: The pull secret obtained from Red Hat OpenShift Cluster Manager at cloud.redhat.com/openshift/install/pull-secret.
+      static_network_config:
+        type: array
+        items:
+          $ref: '#/definitions/agent_static_network_config'
+      image_type:
+        $ref: '#/definitions/image_type'
+      ignition_config_override:
+        type: string
+        description: Json formatted string containing the user overrides for the initial ignition config
+      cluster_id:
+        type: string
+        format: uuid
+        description: If set, all agents that register will be associated with the specified cluster
 
-
-  image_info:
+  infra_env:
     type: object
     properties:
-      ssh_public_key:
+      kind:
+        type: string
+        enum: ['InfraEnv']
+        description: Indicates the type of this object.
+      id:
+        type: string
+        format: uuid
+        description: Unique identifier of the object.
+        x-go-custom-tag: gorm:"primary_key"
+      href:
+        type: string
+        description: Self link.
+      name:
+        type: string
+        description: Name of the InfraEnv.
+      proxy:
+        $ref: "#/definitions/proxy"
+      additional_ntp_sources:
+        type: string
+        description: A comma-separated list of NTP sources (name or IP) going to be added to all the agents.
+      ssh_authorized_key:
         type: string
         description: SSH public key for debugging the installation.
+      pull_secret_set:
+        type: boolean
+        description: True if the pull secret has been added to the cluster.
+      static_network_config:
+        type: string
+        description: static network configuration string in the format expected by discovery ignition generation
+      type:
+        $ref: '#/definitions/image_type'
+      ignition_config_override:
```

Comment:
why do we need ignition_config_override in innfra_env? They are used on during install_config, no?

Reply:
This is for customizing the ignition embedded in the ISO